internal/cli/cli.go

```diff
@@ -115,12 +96,13 @@ func main() {
 		fmt.Println("\nNo rac-files supplied")
 		os.Exit(1)
 	}
-	callback, err := getCallback(*stdout, *outputDirectory, *skipImages, *skipTimeseries)
+	callback, teardown, err := getCallback(*stdout, *outputDirectory, *skipImages, *skipTimeseries)
```

Comment:
it is a mixture of os.Exit, log.Fatal, and return err  in this function. Is this by purpose?

Reply:
No not really, perhaps nice with `log.Fatal` in general